ocrd_eval.sample.yml

```diff
@@ -0,0 +1,85 @@
+- '@id': https://github.com/OCR-D/quiver/tree/data/evaluations/wf1-data345-eval1.json
+  label: OCR workflow 1 on workspace 345
+  metadata:
+    ocr_workflow:
```

Comment:
One suggestion to simplify the structure at this point: 

```
"ocr_workflow": { 
    "@id": "...",
    "label": "....",
    "steps": ["step 1", "step 2", ...]       
}

Reply:
Fine for me, what do you think, @kba ?

Reply:
So, that would replace

```yaml
    workflow_steps:
      '0': Processor A
      '1': Processor B
```

? Sure why not.

Reply:
Yep, that's exactly what we've thought of. I'll add the schema changes.